front/src/utils/date.ts

```diff
@@ -47,6 +47,16 @@ export const dateTimeToIso = (inputDateTime: string) => {
   return null;
 };
 
+export const dateTimeZToIso = (inputDateTime: string) => {
```

Comment:
Should I modify the existing function, or create a new one as I did? Since my argument can't be null, I need to return a string or '',